operations/node_management/NCN_Identify_Drives_Using_ledctl.md

```diff
@@ -0,0 +1,17 @@
+# NCN Drive Identification
+
+Basic usage for the ledmon/ledctl software for drive indentification via the drive leds.
```

Comment:
Change to:

Use the `ledmon`/`ledctl` software to identify NCN drives via the drive LEDs.